pos_hide_partner_info/static/src/xml/templates.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates id="template">
+    <t t-extend="ClientListScreenWidget">
+
+        <t t-jquery="th:contains('Address')" t-operation="attributes">
+            <attribute name="class">oe_hidden</attribute>
+        </t>
+        <t t-jquery="th:contains('Phone')" t-operation="attributes">
+            <attribute name="class">oe_hidden</attribute>
+        </t>
+    </t>
+
+    <t t-extend="ClientLine">
+        <t t-jquery="td:has([t-esc='partner.address'])" t-operation="attributes">
+            <attribute name="class">oe_hidden</attribute>
+        </t>
+        <t t-jquery="td:has([t-esc*='phone'])" t-operation="attributes">
```

Comment:
* typo ?

Reply:
No, it means countains "phone"

Reply:
https://api.jquery.com/category/selectors/

Reply:
oh ! thanks !